### DISCUSSION TITLE
Consumo alimentatori

### DISCUSSION THREAD

**Discussion by kaerry** [indomus/forum#2065] [GENERALE - Discussioni generiche]
Buonasera, chiedo aiuto a voi del gruppo che sicuramente ne sapete più di me..
Ho in casa 5 alimentatori meanwell 240w 24v che mi alimentano tutta l'illuminazione a strip led di casa tramite shelly rgbw2 in modalità white. "Purtroppo" rilevo che con tutte le luci spente in casa ho un consumo medio nella linea luci rilevato tramite pinza amperometrica di circa 160-220 watt, quindi di solo consumo degli alimentatori. Si può fare qualcosa per migliorare la situazione? Pensavo di ridurre gli alimentatori mettendone meno ma più potenti. Grazie per l'attenzione.

**Comment by indomus** [comment]
> Buonasera, chiedo aiuto a voi del gruppo che sicuramente ne sapete più di me.. Ho in casa 5 alimentatori meanwell 240w 24v che mi alimentano tutta l'illuminazione a strip led di casa tramite shelly rgbw2 in modalità white. "Purtroppo" rilevo che con tutte le luci spente in casa ho un consumo medio nella linea luci rilevato tramite pinza amperometrica di circa 160-220 watt, quindi di solo consumo degli alimentatori. Si può fare qualcosa per migliorare la situazione? Pensavo di ridurre gli alimentatori mettendone meno ma più potenti. Grazie per l'attenzione.

240w sono i watt che assorbono al massimo di spinta. con i led spenti non devono assorbire nulla, al massimo 1-2w a unità, per il consumo stesso dello shelly. il consumo che rilevi arriva da un'altra parte.

**Reply by kaerry** [reply]
Non può essere così credo, dal quadro generale parte una fase che alimenta esclusivamente tutti e 5 gli alimentatori e nient'altro.
Misurando con una pinza amperometrica la fase che arriva ad ogni singolo alimentatore vedo che il consumo di ogni apparato va dai 20 ai 30 W con i led spenti, in pratica viene alimentato solamente lo shelly rgbw2.
Mi viene da pensare che gli alimentatori meanwell ELG240 24v non siano per nulla efficienti, o che abbia preso una partita difettosa.

**Reply by indomus** [reply]
non è spiegabile diversamente, perché senza carico applicato non devono assorbire quasi nulla.
